CONTRIBUTING.md

```diff
@@ -18,30 +18,34 @@
 <summary><b>theme example</b></summary>
 
 To create a custom theme you need to define a colorscheme for each of vim's modes. Each mode has a `fg` and `bg` field for every lualine section.
+You can add special effects with `gui`. 
 This is really easy in lua. Here is and example of a gruvbox theme.
 
 ```lua
 local gruvbox = {  }
 
 local colors = {
-  black = "#282828",
-  white = '#ebdbb2',
-  red = '#fb4934',
-  green = '#b8bb26',
-  blue = '#83a598',
-  yellow = '#fe8019',
-
-  gray = '#a89984',
-  darkgray = '#3c3836',
-
-  lightgray = '#504945',
-  inactivegray = '#7c6f64',
+ -- color format { termguicolor, notermguicolor}
+  black = {"#282828", 0},
```

Comment:
These termguicolors need to be changed after we port `gruvbox` theme to support notermguicolors